SimonSaysLab/SimonSaysGame.swift

```diff
@@ -51,9 +51,10 @@ struct SimonSays {
     
     var chosenColors = [Color]()
     var patternToMatch = [Color]()
+    var colorToDisplay = 0
     let numberOfColorsToMatch: Int
     
-    init(numberOfColorsToMatch: Int = 15) {
+    init(numberOfColorsToMatch: Int = 5) {
         self.numberOfColorsToMatch = numberOfColorsToMatch
```

Comment:
Yeah, I found while making the project that 15 was a bit hard to keep in my head.